Globes.js

```diff
@@ -0,0 +1,94 @@
+{
+	"translatorID": "4e9c34ef-89bb-4c7e-97dd-abb61d8b9254",
+	"label": "Globes",
+	"creator": "Eran Rosenthal",
+	"target": "^https?://www\\.globes\\.co\\.il/news/article.aspx",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2019-10-01 10:49:38"
+}
+
+/**
+	Copyright (c) 2019 Eran Rosenthal
+	
+	This program is free software: you can redistribute it and/or
+	modify it under the terms of the GNU Affero General Public License
+	as published by the Free Software Foundation, either version 3 of
+	the License, or (at your option) any later version.
+	
+	This program is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+	Affero General Public License for more details.
+	
+	You should have received a copy of the GNU Affero General Public
+	License along with this program. If not, see
+	<http://www.gnu.org/licenses/>.
+*/
+
+function detectWeb(doc, url) {
+	return 'newspaperArticle';
+}
+function doWeb(doc, url) { 
+	var item = new Zotero.Item('newspaperArticle');
+	
+	item.title = ZU.xpathText(doc, '//meta[@property="og:title"]/@content');
+	item.publicationTitle = 'Globes';
+	item.url = url;
+	item.language = 'he';
+	var abstract = ZU.xpathText(doc, '//meta[@name="description"]/@content');
+	if (abstract) {
+		item.abstractNote = abstract;
+	}
+	
+	var json = ZU.xpathText(doc, '//script[@type="application/ld+json"]');
+	var data = JSON.parse(json);
+	if (data) {
+		
+		if (data.author && Array.isArray(data.author)) {
+			// note only the 0 item is a real author
+			item.creators.push(Zotero.Utilities.cleanAuthor(data.author[0].name, 'author'));
+		}
+		
+		if (data.datePublished) {
+			item.date = ZU.strToISO(data.datePublished);
+		}
+	}
+
+	item.complete();
+}
+/** BEGIN TEST CASES **/
+var testCases = [
+	{
+		"type": "web",
+		"url": "https://www.globes.co.il/news/article.aspx?did=1001302127",
+		"items": [
+			{
+				"itemType": "newspaperArticle",
+				"title": "הקרטל התאילנדי שובר שיאים של אכזריות. האם אדם אחד יצליח לעצור אותם?",
+				"creators": [
+					{
+						"firstName": "ג'ושוע",
+						"lastName": "האמר",
+						"creatorType": "author"
+					}
+				],
+				"date": "2019-01-10",
+				"abstractNote": "גורמים עלומים בצפון-מזרח תאילנד מנהלים כבר שנים מערך מתוחכם ומורכב להפליא של ציד בלתי חוקיים, ומרוויחים הון עתק מהכליה שהם ממיטים על כמה מהמינים האקזוטיים בעולם ● האם האדם היחיד שנאבק בתופעה יצליח להפיל את סינדיקט ההברחות שאיש לא יכול לו?&nbsp;●&nbsp;גיליון שכולו קנאה&nbsp;&nbsp;",
+				"language": "he",
+				"libraryCatalog": "Globes",
+				"publicationTitle": "Globes",
+				"url": "https://www.globes.co.il/news/article.aspx?did=1001302127",
+				"attachments": [],
+				"tags": [],
+				"notes": [],
+				"seeAlso": []
```

Comment:
Can we save section within the newspaper where this article appeared as well? I.e.
```html
<meta property="article:section" content="מגזין G">
```

Reply:
This then should go into the field `section`.

Reply:
Done